.golangci.yml

```diff
@@ -0,0 +1,61 @@
+# This file contains all available configuration options
```

Comment:
Hostname for this test https://github.com/hazelcast/hazelcast-go-client/blob/master/test/ssl/client_ssl_authentication_test.go#L256 should be `foobar.hazelcast.com` due to new comodo certificate